### DISCUSSION TITLE
Wimboot rejects WINPE boot image in pcbios mode

### DISCUSSION THREAD

**Discussion by szy-device** [ipxe/ipxe#1320] [General]
Hello, i have problem with integrating iPXE with PCBIOS mode (in UEFI works perfectly). I am trying boot basic winpe boot.wim image, but nothing is happend, except showing error:
> Found initrd at [...]
> Placing initrd at [...]
> **Bad CPIO magic**
> FATAL: could not extract initrd files

My config from ipxe.menu is:

_kernel http://${server}/Images_Content/wimboot/wimboot gui
initrd -n BCD http://${server}/Images_Content/wimboot/boot/bcd BCD
initrd -n boot.sdi http://${server}/Images_Content/wimboot/boot/boot.sdi boot.sdi
initrd -n boot.wim http://${server}/Images_Content/wimboot/autopilotDetector/boot.wim boot.wim
boot
goto start_

I use ipxe 1.2.0 and wimboot v.2.8.0.
What can I do more to solve that issue? Is it possible?

**Comment by NiKiZe** [comment]
Could you add `imgstat` before `boot` and check that you don't have other files loaded at this stage.

Also to simplify your script, under normal circumstances you can drop the BCD and boot.sdi lines

Your `goto start` will only run if `boot` returns without error, i don't thi knthis will ever happen, but you might want to use `boot || shell` to handle the event that `wimboot` fails.

**Reply by szy-device** [reply]
Thank you for your reply. I tried running according to your directions. With only one `initrd [...] boot.wim`, on the screen I see the command `boot.wim : 364742668 bytes “boot.wim”` and loading stops on Windows Boot Manager with a message, about a missing `BCD` file. Should I successively add other components?

**Reply by NiKiZe** [reply]
What do you mean by "add other components" are you using some installation toolkit to generate?

`boot.wim` from the official MS ISOs already has these files, if your file doesn't you might need to add them separately.

**Reply by szy-device** [reply]
The other components are BCD and boot.sdi. In the end, the solution worked, and the problem was related to outdated component files
